providers/base/units/tracers/resource.pxu

```diff
@@ -0,0 +1,12 @@
+id: kernel_config_file
+plugin: resource
+_summary:
+ Kernel config file resource
+command:
+ if [[ -f "/boot/config-$(uname -r)" ]]
+ then
+     echo "state: supported"
```

Comment:
What's the 'support' in this case?
Looking at what this job does, it seems like it checks whether kernel config file is available in the system.

The id here would suggest, that this resource provides the configuration options of the kernel.

Reply:
Two ways forward from here is either:
1) make the job do what it promises, which is produce all the configuration options with their value, so for instance the ftrace job could require: `kernel_config_file.CONFIG_FTRACE == "y"`, or, simpler:
2) adjust description of the job with what it actually does: check for existence/availability of the kernel config.

Reply:
Description should be more clear now: `_description: Check existence of kernel config file`